docs/norska/porten.md

```diff
@@ -26,8 +26,15 @@ MBus i mätaren agerar master och det är möjligt att ha flera slavar på sling
 lyssnar pushad data. 
 
 
+## Externa referenser
 
-
-
-
+- https://www.nek.no/wp-content/uploads/2018/10/Kamstrup-HAN-NVE-interface-description_rev_3_1.pdf
+- https://www.nek.no/wp-content/uploads/2017/10/AMS-HAN-personvernnotat-h%C3%B8ringsversjon.pdf
+- https://www.energiforetagen.se/globalassets/energiforetagen/det-erbjuder-vi/publikationer/branschrekommendation-lokalt-granssnitt-v1-2-2018.pdf
+- https://www.utomhusliv.se/wp-content/uploads/2020/10/Specifikation-f%C3%B6r-HAN-modulen-f%C3%B6r-elm%C3%A4tare-engelska.pdf
+- https://www.skekraft.se/wp-content/uploads/2021/03/Aidon_Feature_description_RJ45_HAN_Interface_EN.pdf
+- https://www.tekniskaverken.se/siteassets/tekniska-verken/elnat/aidonfd-rj45-han-interface-se-v13a.pdf
+- https://www.kode24.no/guider/smart-meter-part-1-getting-the-meter-data/71287300
+- https://xipher.dk/posts/2020-05-17-using-esp8266-to-monitor-kamstrup-omnipower/
+- https://github.com/Claustn/esp8266-kamstrup-mqtt
```

Comment:
I dont want to link to repos where the owner has not contacted me or I have tried out the solution.